drivers/gpu/drm/tests/drm_framebuffer_test.c

```diff
@@ -83,6 +87,18 @@ static const struct drm_framebuffer_test drm_framebuffer_create_cases[] = {
 		 .pitches = { 4 * MAX_WIDTH, 0, 0 },
 	}
 },
+{ .buffer_created = 0, .name = "ABGR8888 Non-zero buffer offset for unused plane",
+	.cmd = { .width = MAX_WIDTH, .height = MAX_HEIGHT, .pixel_format = DRM_FORMAT_ABGR8888,
+		 .handles = { 1, 0, 0 }, .offsets = { UINT_MAX / 2, UINT_MAX / 2, 0 },
+		 .pitches = { 4 * MAX_WIDTH, 0, 0 }, .flags = DRM_MODE_FB_MODIFIERS,
+	}
+},
```

Comment:
What's the difference between this case and the "ABGR8888 Large buffer offset"?

Reply:
The ABGR8888 pixel format has 1 color plane and the fields `.handles`, `.pitches` and `.offsets` should be set to 0 for the other 2 unused planes (they are arrays where each element represent the value for a color plane, in order). 

In the "ABGR8888 Large buffer offset" case, we have the `.offsets` properly set with some value to the first plane but leaving the other 2 elements zeroed. In this case, I set a non-zero `.offsets` by both the first and the second plane (which is unused by this pixel format), and so it should fail to create the framebuffer.

The "Large" word in the "Large buffer offset" case means that the `.offsets` value used for the valid color plane is just big enough to be considered near to the max value it can be (which depends of the combination of the `.pitches` and `.height` values).

Reply:
Thanks for the explanation!